SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -1847,12 +1847,9 @@ void USpatialNetDriver::TickFlush(float DeltaTime)
 
 		if (SpatialGDKSettings->bBatchSpatialPositionUpdates && Sender != nullptr)
 		{
-			if ((GetElapsedTime() - TimeWhenPositionLastUpdated) >= (1.0f / SpatialGDKSettings->PositionUpdateFrequency))
-			{
-				TimeWhenPositionLastUpdated = GetElapsedTime();
+			TimeWhenPositionLastUpdated = GetElapsedTime();
```

Comment:
Is this variable used for anything now?

Reply:
Actually without the time check is the branch used for anything?

Reply:
That is correct, the variable is not used now that the time check is not happening here anymore and I will remove it. 

Related to the branch itself, I think that this change is basically breaking the batching position updates. Basically, before the change I think what happened was if the removed time check passed, then the updates will be sent without checking for each particular Actor if enough time has elapsed, because it would call `USpatialActorChannel::UpdateSpatialPosition` which only checked that the Actor has travelled enough distance to update, and not `USpatialActorChannel::UpdateSpatialPositionWithFrequencyCheck`. That being said, I remember having a conversation about this particular issue and this piece of the code, and I think that batching position updates will soon be removed and therefore that part of the code as well? I might be wrong about this though.